src/main/resources/db/migration/0350/V399__PlotSurvivalRate.sql

```diff
@@ -0,0 +1,28 @@
+ALTER TABLE tracking.observed_plot_species_totals    ADD COLUMN survival_rate INTEGER;
+ALTER TABLE tracking.observed_subzone_species_totals ADD COLUMN survival_rate INTEGER;
+ALTER TABLE tracking.observed_zone_species_totals    ADD COLUMN survival_rate INTEGER;
+ALTER TABLE tracking.observed_site_species_totals    ADD COLUMN survival_rate INTEGER;
+
+CREATE TABLE tracking.t0_plot
+(
+    id                         BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    monitoring_plot_id         BIGINT NOT NULL REFERENCES tracking.monitoring_plots ON DELETE CASCADE,
+    observation_id             BIGINT REFERENCES tracking.observations ON DELETE CASCADE,
+    species_id                 BIGINT REFERENCES species ON DELETE CASCADE,
+    estimated_planting_density NUMERIC,
+
+    CONSTRAINT t0_plot_null_check CHECK (
+        (observation_id IS NULL AND estimated_planting_density IS NOT NULL AND
+         species_id IS NOT NULL) OR
+        (observation_id IS NOT NULL AND estimated_planting_density IS NULL AND species_id IS NULL)
+    ),
+
+    CONSTRAINT unique_plot_species UNIQUE (monitoring_plot_id, species_id)
+);
+
+CREATE UNIQUE INDEX ON tracking.t0_plot (monitoring_plot_id)
+WHERE observation_id IS NOT NULL;
+
+CREATE INDEX ON tracking.t0_plot (monitoring_plot_id);
```

Comment:
You don't need this index since there's already an index from the unique constraint on `(monitoring_plot_id, species_id)` that the DB can use for this. The other two are needed, though.

Reply:
Ahh that makes sense. I was only thinking about species_id which is not handled, but I understand now.